wordpress-mobile/WordPress-MediaPicker-Android
# Separate the Camera from Device source

This PR separates the camera from the local _device module_ and puts it into a separate module. Right now, if someone wants to use only the camera, their app would need to request all the media access permissions contained in the _device module_. After this change, camera can be requested separately without this limitation.

This is a preparation for a new media source - the [Android photo picker](https://developer.android.com/training/data-storage/shared/photopicker), which will also live in its own module. Google has introduced a new [media permissions policy](https://support.google.com/googleplay/android-developer/answer/13986130), which considers media access dangerous and strongly encourages to use the photo picker. After the new photo picker media source is added, we'll be able to simply replace the current local device source with it, which will comply with the policy change.

**To test:**
1. Start the sample app
2. Try all the different data sources and make sure permission requests and selected image paths are returned as expected
